src/components/Filters/FiltersWithRedux.jsx

```diff
@@ -0,0 +1,214 @@
+import "./Filters.css";
+import { Gridd } from "../Grid/Grid";
+import { useEffect } from "react";
+import { useDispatch, getState } from "react-redux";
+
+
+import {
+  filterByCategory,
+  filterByRating,
+  getAllData,
+  filterLowtoHigh,
+  filterHightoLow,
+} from "../../redux/Slice";
+
+
+const Axios = require("axios");
+
+export const FilterWithRedux = () => {
+  const dispatch = useDispatch();
+
+  //Functions to get data acc. to clicked filter
```

Comment:
comments should only explain 
`why was this piece of  code written`

It should not explain 
`how was this piece of code written` or `what is the meaning of this piece of code`